content/200-concepts/100-components/02-prisma-client/030-crud.mdx

```diff
@@ -922,6 +922,8 @@ await prisma.$transaction([deleteProfile, deletePosts, deleteUsers])
 
 - Doesn't scale as well as having a more generic solution which looks up and `TRUNCATES` your tables regardless of their relational constraints.
 
+> **Note**: This does not apply to the `mongodb` connector but only for relational databases.
```

Comment:
By 'does not apply', does this mean that the `deleteMany` function can't be used on MongoDB?

Reply:
Ah no, I meant that mongo does not suffer from the same scaling issues. So that's something to clarify in that sentence.